src/pybind/mgr/telemetry/module.py

```diff
@@ -186,7 +186,8 @@ def gather_crashinfo(self):
     def compile_report(self):
         report = {
             'leaderboard': False,
-            'report_version': 1
+            'report_version': 1,
+            'report_timestamp': datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f'),
```

Comment:
`datetime.utcnow().isoformat() + 'Z'` ?

Reply:
Are you suggesting this as a different way to format the time?  I was choosing the same format used by Ceph when sending crashdumps

Reply:
isoformat() does look like the same thing, but I then don't understand adding timezone

Reply:
changed to isoformat(), left off the Z